lib/colors.ts

```diff
@@ -128,7 +128,7 @@ export function forceP3(color: Color): P3 {
 }
 
 export function toRgb(color: Color): Rgb {
-  return rgb(clampChroma(color, COLOR_FN))
+  return toGamut('rgb', COLOR_FN)(color)
```

Comment:
Should we call `toGamut('rgb', COLOR_FN)` only once like:

```js
export const = toGamut('rgb', COLOR_FN)
```

It will improve performance

Reply:
I'm not sure about performance because I couldn't find any heavy operations in `toGamut` function, only in the returned function. But it definitely looks much better now. And maybe works faster a little.